scripts/generateMetadata.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+const chalk = require('chalk');
+const prettier = require('prettier');
+
+const reactDocs = require('react-docgen');
+const fs = require('fs');
+const path = require('path');
+
+const root = path.join(__dirname, '../');
+const docsPath = path.join(root, '/docs');
+
+function logError(message) {
+  // eslint-disable-next-line no-console
+  console.log(chalk.red(`❌  Error: ${message}`));
+}
+
+function logSuccess(message) {
+  // eslint-disable-next-line no-console
+  console.log(chalk.green(`✅ ${message}`));
+}
+
+async function docgen(filePath) {
+  const contents = await fs.promises.readFile(filePath, 'utf-8');
+
+  // Not all files have data to parse
```

Comment:
Can we be smarter about which files we're trying to parse? It seems like we can exclude non-JS files, test files, etc?

Reply:
The docgen function filters the files that are are not .js or that are '.test.js'.